domains/rishu.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "DevlopRishi",
+    "email": "gavelrishu@gmail.com"
+  },
+  "records": {
+    "CNAME": "devloprishi.github.io"
+  },
```

```suggestion
  }
```
There shouldn't be a comma here